### DISCUSSION TITLE
SPDM integration of liboqs

### DISCUSSION THREAD

**Discussion by Prashant9800** [open-quantum-safe/liboqs#1607] [Q&A]
I was trying to integrate libspdm with liboqs but finding problems I found one paper and this git repo but build instructions are not very clear can someone help me with this
https://github.com/jyao1/openspdm-pqc/tree/master

**Comment by baentsch** [comment]
> build instructions are not very clear 

[This](https://github.com/jyao1/openspdm-pqc/tree/master/libspdm#prerequisit) seems to be a short but OK description, though dated. That said, this is not work maintained by the OQS team nor on a platform we fully support (Windows), but maybe someone in the community reading here knows about this? Did you already contact its author @jyao1 ?

**Comment by Prashant9800** [comment]
Yes I emailed him awhile back but no response
